server/metrics/metrics.go

```diff
@@ -305,6 +305,9 @@ func New(config *libconfig.MetricsConfig) *Metrics {
 				Name:      collectors.ContainersOOMCountTotal.String(),
 				Help:      "Amount of containers killed because they ran out of memory (OOM) by their name",
 			},
+			// The label `name` can have high cardinality sometimes but it is in the interest of users giving them the
```

Comment:
nit: i think this should go in metrics.md too

Reply:
Updated.